composer.json

```diff
@@ -29,4 +29,4 @@
 			]
 		}
 	}
-}
+}
```

Comment:
Is your editor set to remove new lines at the end of files? Not a blocker but something to check, I have mine set to always add a newline at the end of a file though I think it's just a part of our PHP coding standards.